Frameworks/Foundation/NSDecimalNumber.mm

```diff
@@ -312,34 +350,98 @@ - (NSDecimalNumber*)decimalNumberByRaisingToPower:(NSUInteger)power {
  @Status Stub
  @Notes
 */
-- (NSDecimalNumber*)decimalNumberByMultiplyingByPowerOf10:(short)power {
+- (instancetype)initWithString:(NSString*)numericString locale:(id)locale {
     UNIMPLEMENTED();
     return StubReturn();
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
+*/
+- (NSDecimal)decimalValue {
+    // Read-Only
+    NSDecimal decimal = { 0 };
```

Comment:
#include "stdafx.h"
#include "stdio.h"
#include <iostream>

class Bar
{
    int baz;
};

struct Foo
{
    int a;
    //float a2;
    union B {
        float c;
        long long d[4];
    } b;
    int e;
    Bar bar;
};

void _printFoo(const Foo& f) 
{
    std::cout << "==================\n";
    std::cout << f.a << "\n";
    std::cout << f.b.c << "\n";
    std::cout << f.b.d[0] << "\n";
    std::cout << f.b.d[1] << "\n";
    std::cout << f.b.d[2] << "\n";
    std::cout << f.b.d[3] << "\n";
    std::cout << "==================\n";
}

int main()
{
    Foo f{};
    Foo f2{ 0 };


    _printFoo(f);
    _printFoo(f2);

    return 0;
}

---
In reply to: [80795216](https://github.com/Microsoft/WinObjC/pull/1032#discussion_r80795216) [](ancestors = 80795216,80787871)